### PR TITLE
document Cray support

### PR DESCRIPTION
**Pull request by boegel**
wrapper PR for #205 by @gppezzi, minor update + some formatting changes

### CODE REVIEW

File: docs/Cray-support.rst [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+EasyBuild on Cray
+=================
+
+As of EasyBuild v2.7.0, the support for using EasyBuild on Cray systems is considered stable.
+Tt enables building/installing software using the ``PrgEnv`` modules provided by Cray.
+This page provides an overview of the current status.
+
+For more information about this, contact:
+
+* Guilherme Peretti-Pezzi (CSCS, Switserland, peretti@cscs.ch)
```
Comment (gppezzi) [inline docs/Cray-support.rst]: Switzerland

File: docs/Cray-support.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+*  **HPL** (LINPACK) benchmark version 2.1
+
+Major scientific software applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **CP2K** 2.6.0
+* **GROMACS** 4.6.7
+* **Python** 2.7.9 + numpy 1.9.2 + scipy 0.15.1
+* **WRF** 3.6.1 (pending on Sisu)
+
```
Comment (gppezzi) [inline docs/Cray-support.rst]: How about adding a link to the public [CSCS production repository](https://github.com/eth-cscs/production/) somewhere? There we have an updated list of [supported applications in Piz Daint](https://github.com/eth-cscs/production/blob/master/jenkins-builds/6.0.UP02-2016.11-gpu)

File: docs/Cray-support.rst (hunk 1 above)
Comment (boegel) [inline docs/Cray-support.rst]: urgh, fixed in #286, thanks

File: docs/Cray-support.rst (hunk 2 above)
Comment (boegel) [inline docs/Cray-support.rst]: done in #286

**Pull request merged**